header/src/main/webapp/WEB-INF/jsp/index.jsp

```diff
@@ -284,6 +284,15 @@ if(sec_roles != null) {
             </c:otherwise>
         </c:choose>
 
+        <c:choose>
+            <c:when test='<%= active.equals("mapstore_home") %>'>
+            <li class="active"><a><fmt:message key="mapstore.home"/></a></li>
+            </c:when>
+            <c:otherwise>
+            <li><a href="/mapstore/#home"><fmt:message key="mapstore.home"/></a></li>
```

Comment:
seems like the correct link is /mapstore/#/home (though a redirect is made from /mapstore/#home)